datasets/crass/crass.py

```diff
@@ -21,11 +21,13 @@
 
 
 _CITATION = """\
-@InProceedings{huggingface:dataset,
-title = {A great new dataset},
-author={huggingface, Inc.
-},
-year={2020}
+@misc{frohberg2021crass,
+      title={CRASS: A Novel Data Set and Benchmark to Test Counterfactual Reasoning of Large Language Models}, 
```

Comment:
You introduced a code quality error in line 25: https://circleci.com/gh/huggingface/datasets/57350?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
```
flake8 tests src benchmarks datasets metrics

datasets/crass/crass.py:25:111: W291 trailing whitespace

Reply:
done